### PR TITLE
Add blog for demonstrating the project.

### PR DESCRIPTION
**Pull request by Dhruvi16**
## Description

This PR adds a blog that demonstrates the problem I am trying to solve through my project during Outreachy.

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: content/blog/entries/integartion-vocabulary-ccos/contents.lr [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+open-source
+---
+author: dhruvi16
+---
+pub_date: 2020-07-08
+---
+body:
+
+
+This blog demonstrates the project I am working on during the course of my Outreachy internship. My project involves redesigning and reimplementing one of the web products of the CC network using our new cohesive design library — [Vocabulary](https://cc-vocabulary.netlify.app/?path=/docs/vocabulary-introduction--page). After completion of my internship, the [Creative Commons Open Source](https://opensource.creativecommons.org/) website will have a completely different looking interface that will extensively use Vocabulary components.
```
Comment (hugosolar) [inline content/blog/entries/integartion-vocabulary-ccos/contents.lr:15]:
```suggestion
This blog demonstrates the project I am working on during the course of my Outreachy internship. My project involves redesigning and reimplementing one of the web products of the CC network using our new cohesive design library — [Vocabulary](https://cc-vocabulary.netlify.app/?path=/docs/vocabulary-introduction--page). After my internship is completed, the [Creative Commons Open Source](https://opensource.creativecommons.org/) website will have a complete revamped interface that will extensively use Vocabulary components.
```

File: content/blog/entries/integartion-vocabulary-ccos/contents.lr [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+If we come to the current CC Open Source website, it is quite off-track considering our new design system Vocabulary. The styles and components of the website are not well aligned with the components of the Vocabulary. And thus it lacks harmony and consistency concerning the brand.
+
+The purpose of a design system gets dissolved if it is not well incorporated with the products and we do have this problem with the CC Open Source website. Hence, there was a need for a redesign of CCOS as per the new Design Library. This project aims to rectify the problems and inconsistencies of the CCOS website and build a completely new website (as per Vocabulary).
+
+## The Solution -
+
+I started with trying out different mock-ups for the new website in Figma. You can see what I tried to make [here](https://www.figma.com/file/ka3zs1iYnqJvyLnvAV3cW7/Home-proposals). And then with the help of our UX designer Francisco, we created all the mock-ups and finalized how the website will look in a couple of weeks, here the [link](https://www.figma.com/file/mttcugI1UxvCJRKE5sdpbO/Mockups?node-id=759%3A516) for that.
```
Comment (hugosolar) [inline content/blog/entries/integartion-vocabulary-ccos/contents.lr:27]:
```suggestion
I started with trying out different mock-ups for the new website in Figma. You can see what I tried to make [here](https://www.figma.com/file/ka3zs1iYnqJvyLnvAV3cW7/Home-proposals). And then with the help of our UX designer Francisco, we created all the mock-ups and finalized how the website will look in a couple of weeks, here is the [link](https://www.figma.com/file/mttcugI1UxvCJRKE5sdpbO/Mockups?node-id=759%3A516) for that.
```

File: content/blog/entries/integartion-vocabulary-ccos/contents.lr [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+I started my internship with going through the website’s tech stack, the website is made using Lektor, with which I was not very familiar. I read the official docs went through projects, tried making demo projects, and took help from mentors wherever I got stuck.
```
Comment (hugosolar) [inline content/blog/entries/integartion-vocabulary-ccos/contents.lr:29]:
```suggestion
I started my internship with going through the website’s tech stack, the website is made using [Lektor](https://www.getlektor.com/), with which I was not very familiar. I read the official docs, went through projects, tried making demo projects, and took help from mentors wherever I got stuck.
```

File: content/blog/entries/integartion-vocabulary-ccos/contents.lr [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+To keep the project clean I added a theme that only has templates for the project and alters no content (as per aim). While updating templates, some components were not available in the Vocabulary yet, so I worked on adding those components to the library. And then used them in my new templates. You can see the ongoing work [here](https://cc-open-source-dev.netlify.app/).
```
Comment (hugosolar) [inline content/blog/entries/integartion-vocabulary-ccos/contents.lr:31]: I think to make that Url public could lead to confusion, so I would say let's keep it just for us. Do you have any thoughts on that?

File: content/blog/entries/integartion-vocabulary-ccos/contents.lr [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+I have thoroughly enjoyed contributing to this project as I got to learn so much. And I am excited for what’s next to come in the coming weeks.
```
Comment (hugosolar) [inline content/blog/entries/integartion-vocabulary-ccos/contents.lr:33]:
```suggestion
I have thoroughly enjoyed contributing to this project as I got to learn so much. And I am excited for what’s next to come in the upcoming weeks.
```

**Comment by Dhruvi16** [comment]
I cannot merge this to the master. Can you please do it? @hugosolar

**Comment by hugosolar** [comment]
> I cannot merge this to the master. Can you please do it? @hugosolar

Done 👍

**Pull request merged**